src/main/java/uk/gov/pay/api/model/CardPayment.java

```diff
@@ -42,17 +44,20 @@
     @JsonProperty("provider_id")
     @ApiModelProperty(example = "reference-from-payment-gateway")
     private final String providerId;
+    
+    private final Map<String, Object> metadata;
```

Comment:
I think this should be of type `ExternalMetadata` shouldn't it, since the `metadata` in the connector response has already been deserialized to such? I've been using our custom type everywhere so we're not just using a `Map<String, Object>` which might change in future. There is a custom serializer in Connector that you can reuse.

Sorry I didn't see this on the first pass.